webapp/tsconfig.json

```diff
@@ -9,6 +9,10 @@
     ],
     "module": "ESNext",
     "skipLibCheck": true,
+     "baseUrl": ".",
+    "paths": {
+      "@/*": ["src/*"]
```

Comment:
Not `./src/`? Did you test the build prod?

Reply:
build is ok